test/blackbox/ddsrouter_core/dds/types/Common.hpp

```diff
@@ -12,18 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * @file Common.hpp
```

Comment:
Dear reviewer, would you please suggest a more appropriate name for this file?

Reply:
if it does not contains a class `Common`, better use `common.hpp`. And it could be called `common_interfaces`, or `common_test_utils`, or `test_participants`.